example-application/test/global-setup.js

```diff
@@ -25,12 +24,9 @@ module.exports = async () => {
     );
 
     // ️️️✅ Best Practice: Use npm script for data seeding and migrations
-    const npmLoadAsPromise = util.promisify(npm.load);
-    await npmLoadAsPromise();
-    const npmCommandAsPromise = util.promisify(npm.commands.run);
-    await npmCommandAsPromise(['db:migrate']);
+    execSync('npm run db:migrate');
```

Comment:
I think it is better to run it asynchronously rather than synchronously, running a task that possibly takes time synchronically it's not a good practice.